alembic/versions/12c0ccf6b825_create_trigger.py

```diff
@@ -0,0 +1,98 @@
+"""create trigger
+
+Revision ID: 12c0ccf6b825
+Revises: c8ef88c250e7
+Create Date: 2024-10-27 20:27:30.805137
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '12c0ccf6b825'
+down_revision = 'c8ef88c250e7'
+branch_labels = None
+depends_on = None
+
+create_trigger = """CREATE OR REPLACE FUNCTION log_site_changes()
```

Comment:
This is very nice, and i really like that all the logic is SQL, thanks so much for doing this.

Reply:
No problem! Glad you like it.